README.md

```diff
@@ -1,183 +1,104 @@
-# Tezster 2.0
-A personal development blockchain based on Javascript that lives entirely on your local machine.
+# Tezster-CLI
 
-## Getting Super Powers
+<img src="https://tezster.s3-ap-southeast-1.amazonaws.com/TEZSTER_CLI/1_jDB5enULQVo2UfeiwD32qA.png" alt="Tezster CLI banner" align="center" />
+<div align="center"><strong>A complete testing setup to build, deploy and interact with applications on Tezos sandbox and Tezos testnets.</strong></div>
```

Comment:
A complete toolbox to build, deploy and interact with the applications on Tezos sandbox and Tezos testnets.